pkg/cloud/utils.go

```diff
@@ -24,10 +24,10 @@ import (
 	"github.com/GoogleCloudPlatform/k8s-cloud-provider/pkg/cloud/meta"
 )
 
-const (
-	gaPrefix    = "https://www.googleapis.com/compute/v1"
-	alphaPrefix = "https://www.googleapis.com/compute/alpha"
-	betaPrefix  = "https://www.googleapis.com/compute/beta"
+var (
```

Comment:
Would prefer to not export the variables directly.

var (
  gaPrefix = ..

)

func SetAPIDomain(domain string) {
  gaPrefix = doamin + "/compute/v1"
  ...
}

Reply:
Updated